frontend/alert/src/components/managealert/AlertItem.js

```diff
@@ -60,10 +60,21 @@ export const AlertItem = ({
     return (
         <>
             <GridItem column="1" row={rownum} border halign valign>
-                <input type="checkbox" />
+                <input type="checkbox" checked={checked} onChange={toggleAlert} />
             </GridItem>
             <GridItem column="2" row={rownum} border valign>
-                <P size="0.7rem">{date}</P>
+                {
+                    date
```

Comment:
what date does this represent? from the context I'd guess it has to do with the last time an alert was sent for this registration. Might be better in that case to rename this prop `alertLastSent` or something along that line.